custom_components/powercalc/data/signify/LTP003/model.json

```diff
@@ -4,5 +4,9 @@
    "calculation_strategy": "lut",
    "measure_device":"Gosund SP111 on Tasmota 9.5.0",
    "measure_description":"Measure Script for Tasmota",
-   "measure_method":"script"
+   "measure_method":"script",
+    "aliases": [
+        "LTP001",
```

Comment:
LTP001 seem to be another light model (at least other fixture). It's called Hue Cher.
So this needs to be removed and measured separately.